components/experimental/src/duration/validated_options.rs

```diff
@@ -187,6 +187,12 @@ impl ValidatedDurationFormatterOptions {
             prev_style = *style;
         }
 
+        // section 1.2.1
+        // 27. Set durationFormat.[[FractionalDigits]] to ? GetNumberOption(options, "fractionalDigits", 0, 9, undefined).
+        if let FractionalDigits::Fixed(i) = builder.fractional_digits {
+            builder.fractional_digits = FractionalDigits::Fixed(i.clamp(0, 9));
+        }
```

Comment:
I think `GetNumberOption` returns an error if the number is out of range, not clamping it.

Reply:
On closer reading, `GetNumberOptions` returns the `fallback` value, which in this case is `undefined`. Later uses of fractional digits fall back to default values (`FractionalDigits::ShowAll`). Thanks!